### PR TITLE
Fix Dockerfile to stop buildkite from failing

### PR DESCRIPTION
**Pull request by lyw07**
### Summary

This PR is to stop buildkite from failing. It is failing because `apt-get update` is not run. Dockerfile thinks `apt-get update` is identical in the builds, so it reuses the cache from previous build. Thus, the instruction `apt-get update` is actually not executed.

Reference: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

### Reviewer guidance

Please check the result of this PR's build.

### References

Previous builds for develop were failing on buildkite.

----

### Contributor Checklist

- [X] Contributor has fully tested the PR manually
- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [X] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,27 @@
 FROM ubuntu:xenial
 
 # install latest python and nodejs
-RUN apt-get -y update
-RUN apt-get install -y software-properties-common curl
+RUN apt-get update && apt-get install -y \
+    software-properties-common \
+    curl
```
Comment (benjaoming) [inline Dockerfile:6]: -deleted comment-

**Comment by benjaoming** [comment]
> Dockerfile thinks apt-get update is identical in the builds, so it reuses the cache from previous build.

Can this behavior be switched off entirely or would that be undesirable?

**Comment by lyw07** [comment]
@benjaoming I think we can add `--no-cache=true` option to run `docker build`, but that might increase the build time?

**Comment by benjaoming** [comment]
@lyw07 yup, probably :) I read the docs now and can see that this is how Docker people intended stuff to be..

**Pull request merged**